File: CPP/Baju.cpp
#include "Aksesoris.cpp"

class Baju: public Aksesoris{
	private:
		string Untuk;
		string Size;
		string Merk;
		
	public:
		Baju(){
			
		}
		
		Baju(string untuk, char size, string merk){
			Untuk = untuk;
			Size = size;
			Merk = merk;
		}
		
		// Setter
		void setUntuk(string untuk){Untuk = untuk;}
		void setSize(string size){Size = size;}
		void setMerk(string merk){Merk = merk;}
		
		// Getter
		string getUntuk(){return Untuk;}
		string getSize(){return Size;}
		string getMerk(){return Merk;}
		
		// Fungsi utk menambahkan item
		void addBaju(int id, string nama_produk, string kategori_produk, double harga, string jenis, string bahan, string warna, string untuk, string size, string merk){
			setId(id);
			setNamaProduk(nama_produk);
			setKategoriProduk(kategori_produk);
			setHarga(harga);
			setJenis(jenis);
			setBahan(bahan);
			setWarna(warna);
			setUntuk(untuk);
			setSize(size);
			setMerk(merk);
		}
		
		~Baju(){
			
		}
};

File: PHP/Baju.php
<?php
include "Aksesoris.php";

class Baju extends Aksesoris{
	private $Untuk;
	private $Size;
	private $Merk;
	
	function __construct($untuk = null, $size = null, $merk = null){
		if($untuk !== null && $size !== null && $merk !== null){
			$this->Untuk = $untuk;
			$this->Size = $size;
			$this->Merk = $merk;
		}
	}
	
	// Setter
	function setUntuk($untuk){$this->Untuk = $untuk;}
	function setSize($size){$this->Size = $size;}
	function setMerk($merk){$this->Merk = $merk;}
	
	// Getter
	function getUntuk(){return $this->Untuk;}
	function getSize(){return $this->Size;}
	function getMerk(){return $this->Merk;}
	
	public function addBaju($id, $nama_produk, $kategori_produk, $harga, $jenis, $bahan, $warna, $untuk, $size, $merk, $foto_produk) {
        $this->setId($id);
        $this->setNamaProduk($nama_produk);
        $this->setKategoriProduk($kategori_produk);
        $this->setHarga($harga);
        $this->setJenis($jenis);
        $this->setBahan($bahan);
        $this->setWarna($warna);
        $this->setUntuk($untuk);
        $this->setSize($size);
        $this->setMerk($merk);
		$this->setFotoProduk($foto_produk);
    }
	
	function __destruct(){
		
	}
}

?>